frontend/app/helpers/column-stack.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
```

Comment:
npm 5.3.0
I'll restore its contents from the master branch then. although I'm not sure what I'm doing here